README.md

```diff
@@ -1,12 +1,23 @@
-# Business Site
+# What is this
+My second project during the Technigo bootcamp. 
+Assignment: **Build a Business Site**.
 
-Replace this readme with your own information about your project. 
+## How I started
+After reading/watching/doing all the course material and assignments for week 3 I started by drawing a proper layout for my website. I drew all the different containers and the elements that where meant to be in them. This was something that really helped me to know what to do next and I felt as if I was a few steps ahead all the time. 
+ 
+### What I struggled with and should improve for next weeks project
+I did not manage to understand how HTML media queries for videos work. Neither how to add several sources to one video tag. I spent more than one full day trying different methods but nothing seemed to work. Instead I added the queries to CSS and made 3 different video classes in HTML. Then there was no issue, thankfully. 
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+I should have asked for help instead of going down the google rabbit hole alone. That's something I should improve for next week, asking for help earlier. 
```

Comment:
A good way is to use the stack overflow to ask your questions. If you can help each other you will learn faster, so it's a win win!